app/src/main/kotlin/com/wire/android/ui/userprofile/image/AvatarPickerViewModel.kt

```diff
@@ -44,19 +48,36 @@ class AvatarPickerViewModel @Inject constructor(
             dataStore.avatarAssetId.first()?.apply {
                 avatarRaw = (getUserAvatar(this) as PublicAssetResult.Success).asset
             }
-        } catch (_: ClassCastException) { }
+        } catch (e: ClassCastException) {
```

Comment:
Detekt it's complaining about this empty block, just a logger, or I think a comment like `// NO OP` will do it
https://github.com/wireapp/wire-android-reloaded/runs/5557691317?check_suite_focus=true#step:5:119

Reply:
I added a kalium logger instance on the Wire Application so that we can log these exceptions too